snuba/utils/health_info.py

```diff
@@ -107,7 +107,10 @@ def get_health_info(thorough: Union[bool, str]) -> HealthInfo:
     payload = json.dumps(body)
     if status != 200:
         metrics.increment("healthcheck_failed", tags=metric_tags)
-        logger.error(f"Snuba health check failed! Tags: {metric_tags}")
+        if down_file_exists:
+            logger.error("Snuba health check failed! Tags: %s", metric_tags)
```

Comment:
Moving away from f-strings to logger-native string formatting. The reason we got a new issue at all is because we were interpolating the string before logging. Now we only get one issue instead of N for every possible tag combination.

Reply:
This works because then it's not a "new" issue in Sentry, is that correct?

Reply:
yes that's the goal. now it's grouping by the literal string `"Snuba health check failed! Tags: %s"`